server/api.js

```diff
@@ -192,23 +192,28 @@ router.post("/ablitiy", authorization, async (req, res) => {
 //<------Delete end point from learning objective------>
 router.delete("/learningobjectives/:id", (req, res) => {
   const id = Number(req.params.id);
-      Connection.query(
-        "delete * from learning_objective where id = $1" ,
-        [id],
-        (err, results) => {
-          if (!err) {
-            if(results.rowCount >0){
+  Connection.query(
+    "delete from achievements where learning_obj_id = $1",
+    [id],
+    (err, results) => {
+      if (!err) {
+        Connection.query(
+          "delete from learning_objective where id =$1",
+          [id],
+          (err, results) => {
+            if (!err) {
```

Comment:
This looks ok, but if you want to refactor to `async`/`await` this may make it more readable and less nested/indented